openmc/cell.py

```diff
@@ -618,7 +618,10 @@ def plot(self, *args, **kwargs):
             Axes containing resulting image
 
         """
-        return openmc.Universe(cells=[self]).plot(*args, **kwargs)
+        u = openmc.Universe(cells=[self])
+        ax = u.plot(*args, **kwargs)
+        openmc.release_ids([u.id], openmc.Universe)
+        return ax
```

Comment:
I think the `release_ids` function is a little overkill. We can preserve the original state of the IDs with the following:
```suggestion
        # Create dummy universe but preserve used_ids
        u = openmc.Universe(cells=[self], universe_id=openmc.Universe.next_id + 1)
        openmc.Universe.used_ids.remove(u.id)
        return u.plot(*args, **kwargs)
```